keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -296,11 +294,10 @@ had not yet propagated to all of them.
 
 ### EndpointSlice Controller
 
-When the `TopologyAwareHints` feature gate is enabled and the
-`service.kubernetes.io/topology-aware-routing` annotation is set to `Auto`
-for a Service, the EndpointSlice controller will add hints to EndpointSlices.
-These hints will indicate where an endpoint should be consumed by proxy
-implementations to enable topology aware routing.
+When the `TopologyAwareHints` feature gate is enabled and the `trafficPolicy`
```

Comment:
Intersectional issue - for all of the trafficPolicy values that involve a fallback - what do they fall back to? I'm a bit anxious about this (commented on both KEPs).

Reply:
Good question - wouldn't they fall back to `Cluster` to match the current default behavior?

Reply:
Yeah, but is that what we really want?